### PR TITLE
RavenDB-9784 & RavenDB-11777

### PR DESCRIPTION
**Pull request by ppekrol**

### CODE REVIEW

File: test/SlowTests/Utils/Analyzers/NGramAnalyzer.cs [hunk 1]
```diff
@@ -4,25 +4,43 @@
...
 using Lucene.Net.Analysis.Standard;
-using Lucene.Net.Util;
 using Raven.Server.Documents.Indexes;
+using Raven.Server.Documents.Indexes.Static;
+using Version = Lucene.Net.Util.Version;
 
 namespace SlowTests.Utils.Analyzers
 {
     [NotForQuerying]
     public class NGramAnalyzer : Analyzer
```
Comment (ayende) [inline test/SlowTests/Utils/Analyzers/NGramAnalyzer.cs]: Am I missing something, or is this a test only analyzer?
Shouldn't we be moving this to the server?

**Review by ayende — CHANGES_REQUESTED** [review]

**Comment by ayende** [comment]
Can you also create a full example on how you would create an ngram with specific sizes just for this index (using the new `Index.Configuration` )?

File: test/SlowTests/Utils/Analyzers/NGramAnalyzer.cs (hunk 1 above)
Comment (ppekrol) [reply test/SlowTests/Utils/Analyzers/NGramAnalyzer.cs]: Yes, forgot to move it!

**Comment by ppekrol** [comment]
Yes, but I will wait for AbstractIndexCreationTask.Configuration PR to be merged, it will be easier then

**Comment by ppekrol** [comment]
@ayende moved. You can also check `RavenDB_11777` I'm overriding default NGram's there via Index configuration

**Pull request merged**